ODIN_II/SRC/read_blif.cpp

```diff
@@ -640,6 +640,9 @@ void create_hard_block_nodes(hard_block_models *models, FILE *file, Hashtable *o
   	free_hard_block_ports(ports);
   	mapping_index->destroy_free_items();
 	delete mapping_index;
+	for (i = 0; i < count; i++)
+		vtr::free(mappings[i]);
+		
```

Comment:
move up to line 536

Reply:
the line 540 uses mapping array for "get_hard_block_ports", we can't free them before this line.

Reply:
get_hard_block_ports is at line 535, hence why i think the freeing should be moved ate 536

Reply:
once moved, please remove all the occurence of mappings down here